cli/flags.rs

```diff
@@ -828,13 +828,19 @@ fn compile_subcommand<'a, 'b>() -> App<'a, 'b> {
         .takes_value(true)
         .required(true),
     )
-    .arg(Arg::with_name("out_file").takes_value(true))
+    .arg(
+      Arg::with_name("output")
+        .long("output")
+        .short("o")
+        .help("Output file (defaults to $PWD/<inferred-name>)")
+        .takes_value(true)
+    )
     .about("Compile the script into a self contained executable")
     .long_about(
       "Compiles the given script into a self contained executable.
   deno compile --unstable https://deno.land/std/http/file_server.ts
-  deno compile --unstable https://deno.land/std/examples/colors.ts color_util
-  
+  deno compile --unstable --output color_util https://deno.land/std/examples/colors.ts
```

Comment:
Could you add one more example where you use a fully qualified path?

Reply:
Replaced the existing one, relative path capability is usually obvious.